DEVELOPMENT.md

```diff
@@ -96,21 +98,26 @@ docker run --rm \
 ```
 
 > **NOTE**:
-> * The above command creates a database that persists the data after container shuts down. Specifically, the `-v` flag tells Docker to mount a volume at the path specified.
-> * If you’re running on Windows you’ll need to [change the path for the `-v` option to a Windows-friendly path](https://docs.docker.com/engine/reference/commandline/run/#mount-volume--v---read-only).
+>
+> - The above command creates a database that persists the data after container shuts down. Specifically, the `-v` flag tells Docker to mount a volume at the path specified.
+> - If you’re running on Windows you’ll need to [change the path for the `-v` option to a Windows-friendly path](https://docs.docker.com/engine/reference/commandline/run/#mount-volume--v---read-only).
 
 This creates a Postgres database running in Docker. If you return to your Docker app you can verify that the container is running as expected.
 
 ![Postgres running in Docker](https://user-images.githubusercontent.com/544280/161319618-fe2feab2-0c49-4645-9c8a-57f6c428309b.png)
 
 #### Set environment variables
 
-The Sparrow reference app provides a series of scripts to help you initialize and seed a Postgres database. In order to run, the scripts require two variables to be set on your system.
+The Sparrow reference app provides a series of scripts to help you initialize and seed a Postgres database. In order to run, the scripts require two variables to be set on your system and in your `.env` file and your `.env.local` files.
+
+- `HUB_PROJECTUID`: The UID of the Notehub project that contains your Sparrow data.
+- `DATABASE_URL`: The URL of your Postgres database.
 
-* `HUB_PROJECTUID`: The UID of the Notehub project that contains your Sparrow data.
-* `DATABASE_URL`: The URL of your Postgres database.
+Add both of these variables to your local `.env` and `.env.local` files.
```

Comment:
The big concern here is that the `.env` file is something we have in source control, so even though doing this works, it’s a pretty hacky workaround.

Maybe for now just add a warning to make sure people know `.env` is in source control, and we can clean this up when Carl’s scripts are ready?

Reply:
Good point - I forgot that gets checked in. I'll make a note as you suggest